### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by joepounddev**
PR to test GitHub merging for this project.

• Added database with table for users
• Registration endpoint added

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      directory: "./data/migrations"
+    },
+    seeds: {
+      directory: "./data/seeds"
+    }
+  },
+
+  production: {
+    client: "pg",
+    connection: process.env.DATABASE_URL || {
```
Comment (KevinBrack) [inline knexfile.js:21]: Nice work. Make sure you add a section to the main repo, and store each environment variable you set up. If you use a service that needs an api key to reproduce the project from the source code, add a link to where that can be obtained.

File: middleware/auth/jwt/jwtRestrict.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = (req, res, next) => {
```
Comment (KevinBrack) [inline middleware/auth/jwt/jwtRestrict.js:3]: Custom middleware. That's what I like to see.

File: package.json [hunk 3]
```diff
@@ -5,13 +5,13 @@
...
     "test": "echo \"Error: no test specified\" && exit 1",
-    "server": "nodemon server.js"
+    "server": "nodemon index.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/saltytrolls-bw-may-19/BackEnd.git"
   },
-  "author": "Ajoe Bacus",
+  "author": "Aljoe Bacus",
```
Comment (KevinBrack) [inline package.json:14]: I spelled your name wrong? I'm such a punk...

**Review by KevinBrack — APPROVED** [review]
Looking good so far. Making a ton of progress. Thank you for your hard work. Here if you need anything.

File: package.json (hunk 3 above)
Comment (joepounddev) [reply package.json:14]: Hahaha it's fine, pretty sure it was just a forgotten key press.

**Pull request merged**